Config.yml

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
rename to `config.yml.example`, don't add a config.yml as that could conflict with existing installations.